docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -52,7 +54,7 @@ and time availability), claim it by following these steps:
 
 .. note::
     The Django software foundation requests that anyone contributing more than
-    a trivial change to Django sign and submit a `Contributor License
+    a :ref:`trivial-change`, to Django sign and submit a `Contributor License
```

Comment:
We can update the link text by doing...
```suggestion
    a :ref:`trivial change <trivial-change>`, to Django sign and submit a `Contributor License
```
(the line length will need to be adjusted)